server/parsec/logging.py

```diff
@@ -319,7 +320,8 @@ def enable_sentry_logging(dsn: str, environment: str) -> None:
     sentry_sdk.init(
         dsn=dsn,
         environment=environment,
-        release=__version__,
+        release="parsec@" + __version__,
+        dist=platform.system(),
```

Comment:
Not sure this is useful because this information is already included by Sentry as a tag. But I guess it doesn't do any harm to keep it.